run_go_test.sh

```diff
@@ -15,10 +15,14 @@ trap saveResults EXIT
 
 mkdir "${results_dir}" || true
 
+if [[ -z "${E2E_FOCUS}" ]]; then
+  E2E_FOCUS=".*"
+fi
+
```

Comment:
I'm not sure this is needed. [Reading the docs](https://golang.org/pkg/testing/#hdr-Subtests_and_Sub_benchmarks) I see
```
go test -run ''      # Run all tests.
```

Reply:
Good catch, I will delete the default value.